### PR TITLE
New templates added

### PR DESCRIPTION
**Pull request by pussycat0x**

### CODE REVIEW

File: vulnerabilities/wordpress/wp-iwp-client-listing.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+id: wp-iwp-client-listing
+info:
+  name: WordPress Plugin Iwp-client Listing
+  author: pussycat0x
+  severity: info
+  description: Searches for sensitive directories present in the wordpress-plugins plugin.
+  reference: https://www.exploit-db.com/ghdb/6427
+  tags: wordpress,listing
+requests:
```
Comment (toufik-airane) [inline vulnerabilities/wordpress/wp-iwp-client-listing.yaml:9]:
```suggestion
  tags: wordpress,listing

requests:
```

File: vulnerabilities/wordpress/wp-iwp-client-listing.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  author: pussycat0x
+  severity: info
+  description: Searches for sensitive directories present in the wordpress-plugins plugin.
+  reference: https://www.exploit-db.com/ghdb/6427
+  tags: wordpress,listing
+requests:
+  - method: GET
+    path:
+      - "{{BaseURL}}/wp-content/plugins/iwp-client/"
+    matchers-condition: and
```
Comment (toufik-airane) [inline vulnerabilities/wordpress/wp-iwp-client-listing.yaml:13]:
```suggestion
      - "{{BaseURL}}/wp-content/plugins/iwp-client/"

    matchers-condition: and
```

File: vulnerabilities/wordpress/wp-iwp-client-listing.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    matchers:
+      - type: word
+        words:
+          - "Index of"
+          - "wp-content/plugins/iwp-client/"
+      - type: status
```
Comment (toufik-airane) [inline vulnerabilities/wordpress/wp-iwp-client-listing.yaml:19]:
```suggestion
          - "wp-content/plugins/iwp-client/"

      - type: status
```

File: vulnerabilities/wordpress/wp-sfwd-lms-listing.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+id: wp-sfwd-lms-listing
+info:
+  name: WordPress Plugin Sfwd-lms Listing
+  author: pussycat0x
+  severity: low
+  description: Searches for sensitive directories present in the wordpress-plugins plugin.
+  reference: https://www.exploit-db.com/ghdb/6426
+  tags: wordpress,listing
+requests:
```
Comment (toufik-airane) [inline vulnerabilities/wordpress/wp-sfwd-lms-listing.yaml:9]:
```suggestion
  tags: wordpress,listing

requests:
```

File: vulnerabilities/wordpress/wp-sfwd-lms-listing.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  author: pussycat0x
+  severity: low
+  description: Searches for sensitive directories present in the wordpress-plugins plugin.
+  reference: https://www.exploit-db.com/ghdb/6426
+  tags: wordpress,listing
+requests:
+  - method: GET
+    path:
+      - "{{BaseURL}}/wp-content/plugins/sfwd-lms/"
+    matchers-condition: and
```
Comment (toufik-airane) [inline vulnerabilities/wordpress/wp-sfwd-lms-listing.yaml:13]:
```suggestion
      - "{{BaseURL}}/wp-content/plugins/sfwd-lms/"

    matchers-condition: and
```

File: vulnerabilities/wordpress/wp-sfwd-lms-listing.yaml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    matchers:
+      - type: word
+        words:
+          - "Index of"
+          - "wp-content/plugins/sfwd-lms"
+      - type: status
```
Comment (toufik-airane) [inline vulnerabilities/wordpress/wp-sfwd-lms-listing.yaml:19]:
```suggestion
          - "wp-content/plugins/sfwd-lms"
      - type: status

```

**Review by toufik-airane — CHANGES_REQUESTED** [review]
Dear @pussycat0x,

Thank you for your contribution.
I've made some feedback feel free to accept it.

Nice contribution. 💜

**Review by ehsandeep — APPROVED** [review]

**Review by toufik-airane — COMMENTED** [review]
LGTM 👍

**Pull request merged**